src/Analyzer.Cli.FunctionalTests/ValidTemplateConstants.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
```

Comment:
Minor nit: would be nice to reorder the constants here into the valid and invalid ones, similar to how are separated/ordered in the DataRow attributes in your new test.